mighty/mighty_exploration/mighty_exploration_policy.py

```diff
@@ -8,29 +8,34 @@
 import torch
 from torch.distributions import Categorical, Normal
 
+from mighty.mighty_models import SACModel
+
 
 def sample_nondeterministic_logprobs(
-    z: torch.Tensor, mean: torch.Tensor, log_std: torch.Tensor, sac: bool = False
-) -> Tuple[torch.Tensor, torch.Tensor]:
+    z: torch.Tensor,
+    mean: torch.Tensor,
+    log_std: torch.Tensor,
+    sac: bool = False
```

Comment:
The flag is here to stay model agnostic. Now you make it impossible to add new model classes for SAC...